content/blog/2022-05-16-may-22-heartbeat.md

```diff
@@ -0,0 +1,393 @@
+  [Community Gems.](https://dvc.org/blog/april-22-community-gems) 💎
+
+## ✨ Shiny New Docs
+
+We have a [new doc page](https://dvc.org/doc/start/visualization) showcasing the
+new visualizations added to the
+[example-dvc-experiments repo](https://github.com/iterative/example-dvc-experiments).  
+Whether you need to create plots from tabular data, user-generated plots, or
+autogenerating plots from deep learning code, we've got you covered.
+
+![DVC Visualization Doc](/uploads/images/2022-05-16/dvc-visualization-doc.png '=800')
+_DVC Visualization Doc
+([Source link](https://towardsdatascience.com/an-agile-framework-for-ai-projects-development-cbe115ba86a2))_
+
+## Dmitry Petrov on TFIR about Terraform Provider Iterative (TPI)
+
+[**Dmitry Petrov**](https://twitter.com/FullStackML) recently sat down with
+[**Swapnil Bhartiya**](https://twitter.com/SwapBhartiya) of
+[TFIR](https://www.tfir.io/) to have a chat about TPI. Learn how to save your
+team valuable resources in your machine learning projects with Terraform
+Provider Iterative (TPI). You can watch the recording below.
+
+https://www.youtube.com/watch?v=x-xiKzlQFjY
+
+## 🥳 Join our Release Party Meetup
+
+We have another tool ready to debut on May 24th. On the 25th we'd love to have
+you join us for a Release Party Meetup. We will be celebrating the release of
+the new addition to our open-source tool ecosystem and have a demo of said tool!
+To join the fun,
+[RSVP to the Meetup ](https://www.meetup.com/Machine-Learning-Engineer-Community-Virtual-Meetups/events/285789441/)
+and mark your calendar!
+
+<external-link
+href="https://www.meetup.com/Machine-Learning-Engineer-Community-Virtual-Meetups/events/285789441/"
+title="New Tool Release Party"
+description="Join us May 25th. RSVP for New Tool Release Party!"
+link="https://meetup.com"
+image="/uploads/images/2022-05-16/release-party-meetup.png"/>
+
+## New hires
+
+[**Wolmir Nemitz**]() is our first team member from South America! We're getting
+closer to covering all the continents on
+[our remote team map](https://iterative.ai/about)! From Brazil, Wolmir joins us
+as an Engineer for the 🤫 team (you'll find out June 14th). Wolmir has four
+dogs, two tortoises, and a budgie! 🦜
+
+[**Pavel Chekmaryov**](https://www.linkedin.com/in/ufijuice/) joins us in People
+Operations, managing the hiring pipeline from Frankfurt, Germany, but soon to be
+Canada! He has spent the last eight years in startups, most recently at OccurAI,
+reinventing recruitment in the deeptech/ML field. We look forward to him helping
+to grow our amazing team!
+
+## Open Positions
+
+Even with our amazing new additions to the team, we're still hiring!
+[Use this link](https://iterative.notion.site/Iterative-ai-is-Hiring-852cb978129645e1906e2c9a878a4d22)
+to find details of all the positions and share with anyone you think may be
+interested! 🚀
+
+![Iterative.ai is Hiring](/uploads/images/2022-01-18/hiring.jpeg '=800')
```

Comment:
This image isn't rendering.

Reply:
Interesting. I've used the same code for the past three Heartbeat.  I just moved a copy of the image over. 👍